package/CHANGELOG

```diff
@@ -19,6 +19,7 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * encore.hes() returns non-array details (Issue #2465)
```

Comment:
This is not a fix but a Change because for a user who wants to process the output something changes – please move to Change section.

Reply:
Also add to Changes that hes() does not take the details keyword argument anymore.